blueprints/component-test/index.js

```diff
@@ -22,13 +39,41 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testTypeDefinition = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  // Provide a template (string or precompiled) for this.render()" + EOL +
+      "  this.render();" + EOL + EOL +
```

Comment:
> Maybe a failing test to start isn't so bad?

In some circumstances this might not be so bad, but the problem is just that fixing the failing test requires adding the `ember-template-compiler.js` to the `Brocfile` – or installing the `htmlbars-inline-precompile`.

Also, this would be the first time a generated test would fail ...